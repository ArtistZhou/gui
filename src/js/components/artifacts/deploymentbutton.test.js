import React from 'react';
import renderer from 'react-test-renderer';
import DeploymentButton from './deploymentbutton';

it('renders correctly', () => {
  const tree = renderer.create(<DeploymentButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
