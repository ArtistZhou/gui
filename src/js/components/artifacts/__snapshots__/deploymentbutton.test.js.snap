// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-raised-15"
  disabled={false}
  onBlur={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-2"
  >
    Create deployment
  </span>
  <span
    className="MuiTouchRipple-root-30"
  />
</button>
`;
